cmlreaders/path_finder.py

```diff
@@ -55,9 +55,9 @@ def __init__(self, subject: Optional[str] = None,
         self.experiment = experiment
         self.session = session
 
-        # Stringify localization and montage
-        self.localization = str(localization)
-        self.montage = str(montage)
+        # Stringify localization and montage only if they are given
+        self.localization = (str(localization) if localization is not None else None)
+        self.montage = (str(montage) if montage is not None else None)
```

Comment:
These parentheses are unnecessary

Reply:
Also, shouldn't we default to 0 for these rather than None?

Reply:
Changed it.